src/components/manage/Sidebar/ObjectBrowser.jsx

```diff
@@ -41,12 +52,14 @@ const withObjectBrowser = WrappedComponent =>
       mode = 'image',
       onSelectItem = null,
       dataName = null,
+      propDataName = null,
```

Comment:
Isn't it the same that `dataName`? If it doesn't, can you document it too?

Reply:
no, dataName is the prop inside 'data' object, used for link and image mode.
PropDataName is instead the name of field wich value is 'data'. It's used for 'multiple' mode.

For example:

content:{
'@id': 'page-1',
related_pages:[],
image:{url:""},
link:{href:""}
}

if we use object browser widget for fields:
- related_pages: propDataName is 'related_pages' and dataName is null,
- image: dataName is 'url' and propDataName is null
- link: dataName is 'href' and propDataName is null